packages/shared/src/forms/datePicker/universalDatepicker/DatePicker.ios.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+
+import * as React from 'react';
+import { View, DatePickerIOS } from 'react-native';
```

Comment:
Consider adding the same TODO as for the Android date picker note to make sure we don't forget about this one too?

Reply:
Not sure it matters anymore, I think we are shutting down RN for good.